### DISCUSSION TITLE
Problem with "python3 convert-pth-to-ggml.py models/7B/ 1"

### DISCUSSION THREAD

**Discussion by DIGIX666** [ggml-org/llama.cpp#475] [Q&A]
I have an error when I launch the order : RuntimeError: Internal: unk is not defined

Does anyone know why? 

Because I checked the version of python and the code of the file in question but I can't find

**Comment by j-f1** [comment]
Can you post the stack trace? I don’t see that variable name showing up anywhere in our code.

**Reply by DIGIX666** [reply]
The name of the variable is "is_unknown" in the file "convert-pth-to-ggml.py"

In my first test I had removed the file ggml-vocab.bin from the models folder and I got the error. 
Then I tried again with the ggml-vocab.bin file and the program worked

**Reply by endolith** [reply]
What does `ggml-vocab.bin` do?  I don't see any documentation anywhere.

**Reply by DIGIX666** [reply]
There was no error in the code, I did not accidentally delete a file that should not have been deleted.
